addons/account/models/account_journal.py

```diff
@@ -323,6 +324,44 @@ def _compute_suspense_account_id(self):
             else:
                 journal.suspense_account_id = False
 
+    def _inverse_type(self):
+        for record in self:
+            # Create an alias for purchase/sales journals
+            if record.type in ('purchase', 'sale'):
```

Comment:
You can't return in the middle of a loop
Moreover, it would prevent adding other stuff in there, so a `continue` wouldn't be good either, and it makes the code harder to understand IMO

Reply:
But I did it anyways

Reply:
O yeah, my bad, I meant `continue` ofc.